aten/src/ATen/native/cuda/FunctionOfAMatrixUtilsKernel.cu

```diff
@@ -24,19 +24,26 @@ __global__ void _elemwise_kernel(int total_n_elems, func_t f) {
   }
 }
 
-template <int n_threads, int n_elems_per_thread, typename func_t>
-void _lauch_kernel(int total_n_elems, const func_t& f) {
+template <typename func_t>
+void _launch_kernel(int total_n_elems, const func_t& f) {
   TORCH_INTERNAL_ASSERT(
     total_n_elems >= 0 && total_n_elems <= std::numeric_limits<int32_t>::max()
   );
 
+  const int n_threads = at::cuda::warp_size() * 2;
```

Comment:
In https://github.com/pytorch/pytorch/pull/68008, this value is changed to warpsize * 4